packages/dapp/src/services/wallet.service.ts

```diff
@@ -1,23 +1,37 @@
-import { getStarknet } from "@argent/get-starknet"
+import { connect, getStarknet } from "@argent/get-starknet"
 import { shortString } from "starknet"
 
 import { Network } from "./token.service"
 
-export const isWalletConnected = (): boolean => !!getStarknet()?.isConnected
+export const silentConnectWallet = async () => {
+  const windowStarknetRef = await connect({ showList: false })
```

Comment:
not a ref in the react sense

```suggestion
  const windowStarknet = await connect({ showList: false })
```